### ISSUE TITLE
Add CI tests

### ISSUE THREAD

**Issue by andrewda** [SouthEugeneRoboticsTeam/BunnyBots-2019#14]
It's a good idea to build the robot code on Travis CI or GitHub Actions (see below) so that you can verify the code is working before merging it. Take a look at https://github.com/SouthEugeneRoboticsTeam/DeepSpace-2019/blob/development/.travis.yml for reference.

## Steps

- [ ] Add a `.travis.yml` file according to [the link above](https://github.com/SouthEugeneRoboticsTeam/DeepSpace-2019/blob/development/.travis.yml), or check out the [GitHub Actions reference](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions)
- [ ] Create a pull request with the changes
- [ ] Make sure that [a build has started](https://travis-ci.com/SouthEugeneRoboticsTeam/BunnyBots-2019/) on Travis

**Comment by mckelveygreg** [comment]
I've been experimenting with Github actions at work! I found it nice not to have to manage another account, plus there is a free plan for private repos
https://github.com/marketplace/actions/gradle-command

**Comment by andrewda** [comment]
@mckelveygreg Good point, I've also had very good experiences with GitHub actions! Might be a good thing to look into for the robot
